desktop/assets/texts/ru-RU/preferences_dialog.ftl

```diff
@@ -23,3 +23,8 @@ theme = Тема
 theme-system = Как в системе
 theme-light = Светлая
 theme-dark = Тёмная
+# See for context https://github.com/FeralInteractive/gamemode
+gamemode = GameMode
+gamemode-tooltip = GameMode темпоральною добавит сети оптимизации в твоем компьютере и/или в Ruffle. Ruffle просит GameMode только когда фильм играет.
```

Comment:
Google Translate translates this as "network optimizations", but what does it know...